api-testing/functional/tests/cypress/integration/01-institution/patch-institutions-institution.ts

```diff
@@ -0,0 +1,49 @@
+let identity: any;
+describe("Institution api", () => {
+  it("POST - Token", () => {
+    cy.request({
+      method: "POST",
+      url: Cypress.env("token_url"),
+      form: true,
+      body: {
+        grant_type: Cypress.env("grantType"),
+        client_id: Cypress.env("clientId"),
+        username: Cypress.env("userName"),
+        password: Cypress.env("password"),
+      },
+    }).then((response) => {
+      identity = response;
+      window.localStorage.setItem("identity", JSON.stringify(identity));
+    });
+  });
+
+  it("PATCH - Update institution details", () => {
+    cy.fixture("bodyData").then((data) => {
+      cy.request({
+        method: "PATCH",
+        url: Cypress.env("update_institution_url"),
+        auth: { bearer: identity.body.access_token },
```

Comment:
What is the plan to consume the token? I saw that is set to the storage but it is consumed from the previous method response.
The cypress tests are going to run always sequentially or the car run in parallel?
Here there is a dependency from the first method to run in order for the second to be executed.

Reply:
It will run once before every tests. If there are more than one test case then will implement this function.

![image](https://user-images.githubusercontent.com/98920571/170768884-7a2145a4-ca60-4474-84d3-7b7dddf3da8f.png)

Reply:
Before any test is run, the token method will run.
<img width="462" alt="image" src="https://user-images.githubusercontent.com/98920571/171024439-d70b0280-5da3-
4b94-9620-2d14d246dea2.png">